db/migrate/20210719031121_remove_alert_topics_index.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class RemoveAlertTopicsIndex < ActiveRecord::Migration[6.1]
```

Comment:
I guess this should be a post-migrate? Otherwise the `CreateAlertTopicsIndex` might run again and recreate the index?